src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32.cs

```diff
@@ -292,26 +290,24 @@ private static Win32 InternetShortcutProgram(string path)
             string[] lines = System.IO.File.ReadAllLines(path);
             string appName = string.Empty;
             string iconPath = string.Empty;
+            string urlPath = string.Empty;
             string scheme = string.Empty;
             bool validApp = false;
 
-            const string steamScheme = "steam";
+            Regex urlSchemes = new Regex(@"^steam:\/\/rungameid\/|^com\.epicgames\.launcher:\/\/apps\/");
+
             const string urlPrefix = "URL=";
             const string iconFilePrefix = "IconFile=";
-            const string hostnameRun = "run";
-            const string hostnameRunGameId = "rungameid";
 
             foreach(string line in lines)
             {
                 if(line.StartsWith(urlPrefix))
                 {
-                    var urlPath = line.Substring(urlPrefix.Length);
+                    urlPath = line.Substring(urlPrefix.Length);
                     Uri uri = new Uri(urlPath);
 
                     // To filter out only those steam shortcuts which have 'run' or 'rungameid' as the hostname
-                    if(uri.Scheme.Equals(steamScheme, StringComparison.OrdinalIgnoreCase)
-                        && (uri.Host.Equals(hostnameRun, StringComparison.OrdinalIgnoreCase)
-                        || uri.Host.Equals(hostnameRunGameId, StringComparison.OrdinalIgnoreCase)))
+                    if(urlSchemes.Match(urlPath).Success)

```

Comment:
run and rungameid are not schemes, rather they are hostnames. Can the variable be renamed?

Reply:
I was trying to match the entire prefix `steam://rungameid` and not just hostname i.e `rungameid`. So naming it `hostname` didn't seem to be correct. How about I rename this to `URLPrefixes`?

Reply:
`URLPrefixes` sounds good.